tests/config.py

```diff
@@ -36,6 +36,9 @@
 CLUSTER_PREFIX = os.getenv('CLUSTER_PREFIX', 'rookcheck-%s-%s-' % (
     getpass.getuser(), datetime.datetime.today().strftime('%Y%m%d')))
 
+NB_MASTERS = os.getenv('NB_MASTERS', 1)
```

Comment:
NB? What does that mean? I would prefer to prefix the rookcheck env vars with `ROOKCHECK_`

Reply:
I think I would use K8S then because this is more related to Kubernetes than to rook_check ?

Reply:
Like `K8S_MASTERS`

Reply:
But it's only relevant for rookcheck. so if you type `env|grep -i rookcheck` you get the relevant config options

Reply:
I'm not sure I'm following. Rookcheck is the whole project so everything is related to rookcheck.
For example WORKSPACE_DIR and CLUSTER_PREFIX are related to ROOKCHECK process more closely, but other vars really do configure provides overall.

I mean I'm ok to do as you propose but I don't see the logic.

Reply:
I'm in favor of changing `WORKSPACE_DIR` and friends, too. There are eg OS_* vars that are related to openstack libcloud. so not everything is rookcheck